azure-pipelines.yml

```diff
@@ -194,3 +194,24 @@ jobs:
 
   - script: python setup.py sdist
     displayName: 'Build sdist'
+
+
+- job: DocsTest
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+  - bash: echo "##vso[task.prependpath]/usr/share/miniconda/bin"
+    displayName: Add conda to PATH
+  - script: conda env create -n test_env --quiet --file ci/doc.yml
+    displayName: Create Anaconda environment
+  - script: |
+      source activate test_env
+      # Replicate the exact environment created by the readthedocs CI
+      # conda install --yes --quiet -c pkgs/main mock pillow sphinx sphinx_rtd_theme
+      pip install -e .[all]
```

Comment:
maybe just `pip install .[docs]` on top of the python3.8 environment?

Reply:
rtd uses this file to manage its environment https://github.com/pvlib/pvlib-python/blob/master/readthedocs.yml

Reply:
I tried using just .[docs] in a previous commit but that was unsuccessful. The RTD file is helpful to confirm that using .[all] is necessary. Do you think it's worth it's own issue to address that the .[docs] is insufficient to build?

Reply:
some of the doc pages, e.g. [Forecasting](https://pvlib-python.readthedocs.io/en/stable/forecasts.html), need the optional dependencies. I don't know what the right approach is long term, but I agree it's better to use all for now.